lib/middleware.js

```diff
@@ -50,6 +50,13 @@ class Kevin {
                 requestPath.replace(/^\//, "").replace(/\.js$/, ""),
             // Only build assets; don't handle serving them. This is useful if you want to do
             // something with the built asset before serving it.
+            selectConfigNameGivenReqPath = (reqPath, configNames) => {
```

Comment:
this name is precise, but a little clunky for a user-facing API imo. how about shortening it to `selectConfigName`?

Reply:
done